app/retail/emails.py

```diff
@@ -948,14 +948,15 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-Our first hackathon of the summer season is here. Beyond Blockchain, a hackathon organized alongside Consensys Labs, focuses on the future. Over $15k of prizes will be available for this three week hackathon, and the hacking begins on Monday, June 24th, and goes until July 10th.
-For more info, read the announcement blog post <a href="https://gitcoin.co/blog/beyond-blockchain-hackathon/" target="_blank"> here </a> and join our Discord <a href="https://discord.gg/T98dQYC" target="_blank"> here! </a>
+Beyond Blockchain has officially begun! We kicked off the three week haul on Monday, and there's still plenty of time to vie for some of the prizes we have available. Our sponsors -- Consensys Labs, Raiden, Arweave, Portis, and The Graph -- have dozens of prizes ranging up to 22 ETH. Join our <a href="https://docs.google.com/document/d/1N8EcHOP7pzuvoPNvJ3VPBtv9KFJf9Ot1kjGU381Dh_s/edit?usp=sharing" target="_blank">Discord</a> and discuss the hackathon,
```

Comment:
what is `https://docs.google.com/document/d/1N8EcHOP7pzuvoPNvJ3VPBtv9KFJf9Ot1kjGU381Dh_s/edit?usp=sharing` ? that doesnt seem like a link to the discord.  in fact, its a permissioned gdoc